src/init.js

```diff
@@ -1,12 +1,65 @@
-import Vue from "vue";
+import { createApp } from "vue";
+import { createRouter, createWebHistory, createWebHashHistory } from "vue-router";
 import StacBrowser from "./StacBrowser.vue";
 import i18n, { loadDefaultMessages } from './i18n';
+import CONFIG from './config';
+import getRoutes from "./router";
+import getStore from "./store";
+
+import { 
+  AlertPlugin, BadgePlugin, ButtonGroupPlugin, ButtonPlugin,
+  CardPlugin, FormPlugin, FormGroupPlugin, FormInputPlugin, FormCheckboxPlugin, FormRadioPlugin,
+  LayoutPlugin, ListGroupPlugin, PopoverPlugin, SpinnerPlugin,
+  VBToggle, VBVisible } from "bootstrap-vue";
+import "bootstrap/dist/css/bootstrap.css";
+import "bootstrap-vue/dist/bootstrap-vue.css";
 
 export default function init() {
   return loadDefaultMessages().then(() => {
-    return new Vue({
-      i18n,
-      render: h => h(StacBrowser)
-    }).$mount("#stac-browser");
+    // Setup router
+    const router = createRouter({
+      history: CONFIG.historyMode === 'history' ? createWebHistory(CONFIG.pathPrefix) : createWebHashHistory(CONFIG.pathPrefix),
+      routes: getRoutes(CONFIG),
+      scrollBehavior: (to, from, savedPosition) => {
+        if (to.path !== from.path) {
+          return { x: 0, y: 0 };
+        }
+        else {
+          return savedPosition;
+        }
+      }
+    });
```

Comment:
We should check whether the scrollBehavior workaround is still needed in the new vue-router.